src/test/java/org/folio/edge/sip2/parser/BlockPatronMessageParserTests.java

```diff
@@ -6,18 +6,21 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
```

Comment:
Eclipse warns, import not used...

Instead of commenting on all the imports not used, I'll just add this one here. Many classes in this package have imports that are not used.

Here is the list:
```
Description	Resource	Path	Location	Type
The import java.time.ZoneId is never used	BlockPatronMessageParserTests.java	/edge-sip2/src/test/java/org/folio/edge/sip2/parser	line 9	Java Problem
The import java.time.ZoneId is never used	CheckinMessageParserTests.java	/edge-sip2/src/test/java/org/folio/edge/sip2/parser	line 10	Java Problem
The import java.time.ZoneOffset is never used	BlockPatronMessageParserTests.java	/edge-sip2/src/test/java/org/folio/edge/sip2/parser	line 10	Java Problem
The import java.time.ZoneOffset is never used	CheckinMessageParserTests.java	/edge-sip2/src/test/java/org/folio/edge/sip2/parser	line 11	Java Problem
The import java.time.ZoneOffset is never used	CheckoutMessageParserTests.java	/edge-sip2/src/test/java/org/folio/edge/sip2/parser	line 10	Java Problem
The import java.time.ZoneOffset is never used	EndPatronSessionMessageParserTests.java	/edge-sip2/src/test/java/org/folio/edge/sip2/parser	line 8	Java Problem
The import java.time.ZoneOffset is never used	FeePaidMessageParserTests.java	/edge-sip2/src/test/java/org/folio/edge/sip2/parser	line 11	Java Problem
The import java.time.ZoneOffset is never used	HoldMessageParserTests.java	/edge-sip2/src/test/java/org/folio/edge/sip2/parser	line 11	Java Problem
The import java.time.ZoneOffset is never used	ItemInformationMessageParserTests.java	/edge-sip2/src/test/java/org/folio/edge/sip2/parser	line 9	Java Problem
The import java.time.ZoneOffset is never used	ItemStatusUpdateMessageParserTests.java	/edge-sip2/src/test/java/org/folio/edge/sip2/parser	line 9	Java Problem
The import java.time.ZoneOffset is never used	PatronEnableMessageParserTests.java	/edge-sip2/src/test/java/org/folio/edge/sip2/parser	line 8	Java Problem
The import java.time.ZoneOffset is never used	PatronInformationMessageParserTests.java	/edge-sip2/src/test/java/org/folio/edge/sip2/parser	line 16	Java Problem
The import java.time.ZoneOffset is never used	RenewAllMessageParserTests.java	/edge-sip2/src/test/java/org/folio/edge/sip2/parser	line 9	Java Problem
The import java.time.ZoneOffset is never used	RenewMessageParserTests.java	/edge-sip2/src/test/java/org/folio/edge/sip2/parser	line 10	Java Problem
The import org.folio.edge.sip2.utils.Utils is never used	BlockPatronMessageParserTests.java	/edge-sip2/src/test/java/org/folio/edge/sip2/parser	line 15	Java Problem
```

Reply:
I may have to start using Eclipse to save myself from embarrassments because my not-so-intelligent IntelliJ doesn't show this list. [](https://intellij-support.jetbrains.com/hc/en-us/community/posts/115000033264-View-All-Project-Errors-Warnings) However, running IntelliJ code inspection reveals over 500 issues 
![image](https://user-images.githubusercontent.com/33640969/57780297-f1f5e580-76f5-11e9-86a7-4b97fab5f792.png).  I think we need to discuss what is the authority, what is needed for the code to be in a good shape. If we follow a particular IDE's suggestions, we may never get to release!

Reply:
Interesting. I'd be curious what these warnings amount to. 695 typos in "Spelling" is interesting. 😄

Is there a way to export this list?